### PR TITLE
Issue 102

### PR DESCRIPTION
**Pull request by jsstevenson**

### CODE REVIEW

File: therapy/schemas.py [hunk 1]
```diff
@@ -152,7 +153,6 @@ class ProhibitedSources(Enum):
     """
 
     CHEMBL = SourceName.CHEMBL.value
```
Comment (jsstevenson) [inline therapy/schemas.py:155]: Wasn't sure if it was worth keeping this now that it only has one member

Comment (korikuzma) [reply therapy/schemas.py:155]: I think it's fine to keep

File: therapy/etl/drugbank.py [hunk 2]
```diff
@@ -67,295 +31,79 @@ def perform_etl(self) -> List[str]:
...
+                if label:
+                    params['label'] = label
+                aliases = [
+                    a for a in row[1].split(' | ') + row[5].split(' | ') if a
+                ]
+                if aliases:
+                    params['aliases'] = aliases
+                cas_ref = row[3]
+                if cas_ref:
+                    params['other_identifiers'] = [
```
Comment (korikuzma) [inline therapy/etl/drugbank.py:105]: Should we be adding UNII and Standard InChI Key to `other_identifiers`?

File: tests/conftest.py [hunk 3]
```diff
@@ -134,3 +134,10 @@ def compare_records(actual: Drug, fixture: Drug):
     assert actual.approval_status == fixture.approval_status
     assert set(actual.other_identifiers) == set(fixture.other_identifiers)
     assert set(actual.xrefs) == set(fixture.xrefs)
+
+
+def compare_response(response: Dict, fixture: Drug, match_type: MatchType):
+    """Check that test response is correct."""
+    assert response['match_type'] == match_type
+    assert len(response['records']) == 1
```
Comment (korikuzma) [inline tests/conftest.py:142]: This may not always be one (I think). We should add a param for this and make the default 1.

**Review by korikuzma — APPROVED** [review]

File: therapy/etl/drugbank.py (hunk 2 above)
Comment (jsstevenson) [reply therapy/etl/drugbank.py:105]: I assumed Alex had a reason to not include them in the first pass -- I'll ask him tomorrow

**Pull request merged**